include/z64scene.h

```diff
@@ -178,16 +178,21 @@ typedef struct Room {
 typedef struct RoomContext {
     /* 0x00 */ Room curRoom;
     /* 0x14 */ Room prevRoom;
-    /* 0x28 */ void* bufPtrs[2];
-    /* 0x30 */ u8 unk_30;
-    /* 0x31 */ s8 status;
-    /* 0x34 */ void* unk_34;
+    /* 0x28 */ void* bufPtrs[2]; // Start and end pointers for the room buffer. Can be split into two pages, where page 0 is allocated from the start pointer and page 1 is allocated from the end pointer.
+    /* 0x30 */ u8 activeBufPage; // 0 - First page in memory, 1 - Last page in memory
+    /* 0x31 */ s8 status; // 0 = Free for new room request, 1 = DmaRequest for a new room is in progress
```

Comment:
super nit: `// 0 = Free ...` -> `// 0 - Free ...` is more consistent with the comment above